### ISSUE TITLE
poweroff or undeploy not completely turning off vcd_vapp

### ISSUE THREAD

**Issue by sgutermann** [vmware/ansible-module-vcloud-director#132]
There was an earlier issue for the vcd_vapp_vm which seemed to be handled. Looks like this was implemented for vcd_vapp as well, but not funtioning there.

 - name: power off vapp
   vcd_vapp:
    vapp_name: "test_vapp"
    vdc: "test_vdc"
    operation: "poweroff"

poweroff simply turns the whole vapp off as well as the vms inside

I have no idea what "undeploy" does as it simply states that the vapp is undeployed, but in fact it is still running as well as all vms inside.

**Comment by sgutermann** [comment]
I am not sure if I am clear on what I am looking for. Just to be sure. The end result that I am looking for is that the whole vapp and vms are turned off for real.

Currently all I can achive is a "stopped" status or a "powered off" status which is not totally off as I can still go to the v-cloud menu and then "tunroff" the vapp.

Now you ask "who cares", well if your provider charges you for something you turned off (but not completely) this simple point does get more interesting.

**Comment by sgutermann** [comment]
Sorry if this seems like spamming, But this is not leeting go of me. I just installed and tested the VM Ware vcd-cli just to see if they had the same problem. Well they do not, but they are confusing in terms of terminology, at least to me.

so if you do:

vcd vapp power-off VAPPNAME

then this does a power-off just the way our ansible module does here, in other words it turns it off but not completely, but if you do a:

vcd vapp stop VAPPNAME

this indeed does turn everything off completely

I am not sure if this helps solve our problem here, but I found it interesting in terms of strange terminology at least in my opinion
